numba/unicode.py

```diff
@@ -541,6 +541,73 @@ def find_impl(a, b):
         return find_impl
 
 
+@overload_method(types.UnicodeType, 'rfind')
+def unicode_rfind(s, substr, start=None, end=None):
+    """Implements str.rfind()"""
+    def unicode_rfind_check_type(ty, name):
+        """Check object belongs to one of specific types
+        ty: type
+            Type of the object
+        name: str
+            Name of the object
+        """
+        thety = ty
+        # if the type is omitted, the concrete type is the value
+        if isinstance(ty, types.Omitted):
```

Comment:
Did you manage to call the function in such a way that this case actually happened? I am trying to construct an example where this would be the case.

Reply:
Debug printing proves that the branch is covered by the tests, e.g. by `test_rfind`.

Reply:
o.k. It would be good to figure out how to force this.